notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q "openvino-dev&gt;=2023.1.0"
</pre><p>Can we install openvino package instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
fixed!